docs/user-guide/global-configurations/pull-image-digest.md

```diff
@@ -0,0 +1,52 @@
+# Pull Image Digest
+
+## Introduction
+
+Devtron offers the option to [pull container images using digest](../creating-application/workflow/cd-pipeline.md#pull-container-image-with-image-digest). Though it can be enabled by an application-admin for a given CD Pipeline, Devtron also allows super-admins to enable pull image digest at environment level.
+
+This helps in better governance and less repetitiveness if you wish to manage pull image digest for multiple applications across environments.
```

Comment:
What this feature actually does? What are the drawbacks? Why should I enable this?

Reply:
Correct. All of these were already explained in the CD Pipeline doc. The hyperlink was given, but now the line has been replaced and made more prominent as follows: _Refer [CD Pipeline - Image Digest](../creating-application/workflow/cd-pipeline.md#pull-container-image-with-image-digest) to know the purpose it serves._